src/unix/freebsd/disk.rs

```diff
@@ -164,19 +162,32 @@ impl GetValues for DiskInner {
     }
 }
 
-fn refresh_disk(disk: &mut DiskInner) -> bool {
-    unsafe {
-        let mut vfs: libc::statvfs = std::mem::zeroed();
-        if libc::statvfs(disk.c_mount_point.as_ptr() as *const _, &mut vfs as *mut _) < 0 {
-            return false;
-        }
-        let block_size: u64 = vfs.f_frsize as _;
+fn refresh_disk(disk: &mut DiskInner, refresh_kind: DiskRefreshKind) -> bool {
+    if refresh_kind.details() {
+        let (total_space, available_space) = unsafe {
+            let mut vfs: libc::statvfs = std::mem::zeroed();
+            if libc::statvfs(disk.c_mount_point.as_ptr() as *const _, &mut vfs as *mut _) < 0 {
+                return false;
```

Comment:
Should not return here. However it should likely use `sysinfo_debug!`.

Reply:
Done